lib/rules/jsx-first-prop-new-line.js

```diff
@@ -35,20 +36,27 @@ module.exports = {
           (configuration === 'multiline-multiprop' && isMultilineJSX(node) && node.attributes.length > 1) ||
           (configuration === 'always')
         ) {
-          node.attributes.forEach(function(decl) {
+          node.attributes.some(function(decl) {
             if (decl.loc.start.line === node.loc.start.line) {
               context.report({
                 node: decl,
-                message: 'Property should be placed on a new line'
+                message: 'Property should be placed on a new line',
+                fix: function(fixer) {
+                  return fixer.replaceTextRange([node.name.end, decl.start], '\n');
+                }
               });
             }
+            return true;
```

Comment:
yeah, its somewhat cleaner than adding `&& node.attributes.length > 0`. Matter of preference.